new-frontiers-14/frontier-station-14
# Adds an innate mineral scanner to dwarves

## About the PR



What it says on the tin.  Dwarves get a 5m innate mineral scanner (equivalent to the base mineral scanner, still inferior to the advanced version).

## Why / Balance


Dwarves are one of the more bland species in-game.  This gives them a nice little something that doesn't seem too gamey (though opinions may vary).

## How to test


1. Join as a dwarf.
2. Spawn a few asteroids with ore.
3. Equip an advanced mining scanner.
4. Your range should increase to 10 meters (check MiningScannerViewerComponent via VV)
5. Unequip the mining scanner.
6. Your range should revert to 5 meters (again, check MiningScannerViewerComponent via VV)
7. Re-equip, unequip, re-equip, unequip, you should always have the mining scanner viewer active.

## Media


## Requirements



## Breaking changes


**Changelog**


:cl:
- add: Dwarves now have an innate mineral scanner that senses ore out to 5 meters.
